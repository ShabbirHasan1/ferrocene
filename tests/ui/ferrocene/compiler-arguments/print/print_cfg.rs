// check-pass
// compile-flags: --print cfg
// only-x86_64-unknown-linux-gnu
// compare-output-lines-by-subset

fn main() {}

// ferrocene-annotations: um_rustc_print
