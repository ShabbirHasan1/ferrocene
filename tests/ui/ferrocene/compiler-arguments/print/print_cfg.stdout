debug_assertions
panic="unwind"
target_abi=""
target_arch="x86_64"
target_endian="little"
target_env="gnu"
target_family="unix"
target_feature="fxsr"
target_feature="llvm14-builtins-abi"
target_feature="sse"
target_feature="sse2"
target_has_atomic="16"
target_has_atomic="32"
target_has_atomic="64"
target_has_atomic="8"
target_has_atomic="ptr"
target_has_atomic_equal_alignment="16"
target_has_atomic_equal_alignment="32"
target_has_atomic_equal_alignment="64"
target_has_atomic_equal_alignment="8"
target_has_atomic_equal_alignment="ptr"
target_has_atomic_load_store="16"
target_has_atomic_load_store="32"
target_has_atomic_load_store="64"
target_has_atomic_load_store="8"
target_has_atomic_load_store="ptr"
target_os="linux"
target_pointer_width="64"
target_thread_local
target_vendor="unknown"
unix
